services/src/handlers/wfs.rs

```diff
@@ -29,15 +28,15 @@ pub(crate) fn wfs_handler<C: Context>(
 ) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
     warp::get()
         .and(warp::path!("wfs"))
+        .and(authenticate(ctx))
```

Comment:
Oh, so you just added WFS authentication in this PR?

Reply:
if yes then this should also be an extra PR

Reply:
I needed it because I require a session to have a user attached in order to create an execution context. That's why I added it...